docrepr/sphinxify.py

```diff
@@ -203,10 +203,11 @@ def generate_extensions(render_math):
     """Generate list of Sphinx extensions"""
     # We need jsmath to get pretty plain-text latex in docstrings
     extensions = []
-    if sphinx.__version__ < "1.1" or not render_math:
```

Comment:
`if sphinx.__version__ < "1.1" or not render_math` seems to fail later if `render_math` is False: The reason is that latest sphinx versions seem to have removed jsmath, though we try to load it when `render_math` is False.

Reply:
Makes sense, thanks!